src/tests/JIT/opt/ObjectStackAllocation/ObjectStackAllocationTests.csproj

```diff
@@ -16,9 +16,5 @@
     <CLRTestEnvironmentVariable Include="DOTNET_JitNoForceFallback" Value="1" />
```

Comment:
`DOTNET_ProfApi_RejitOnAttach` looks unnecessary too. No other codegen test does that.

Reply:
Should this test be marked as `<JitOptimizationSensitive>true</JitOptimizationSensitive>` instead of forcing a random random set of env variables?

Reply:
> Should this test be marked as `<JitOptimizationSensitive>true</JitOptimizationSensitive>` instead of forcing a random random set of env variables?

Maybe? I presume the idea was to still be able to run it under JitStress. But since StackAllocObject is now enabled everywhere I guess JitStress can kick in in other places